source/fab/config.ini

```diff
@@ -0,0 +1,8 @@
+[settings]
+target =
+exec-name =
+
+[flags]
+fpp-flags =
+fc-flags =
+ld-flags =
```

Comment:
Is this file left over from debug?

Reply:
Yes it was. However now I'm wondering, related to your later question about a zero configuration mode of operation, if thee should be a default configuration file here? Or if I should have the configparser library build one from default values if one isn't present.

Reply:
I think there are two modes of operation for zero configuration mode. The first is to simply build what is possible by making assumptions regarding what is to be built and how to do it. The second is to generate as much configuration file as possible using the same heuristics as the basis for a configuration driven build.

We probably don't want to get stuck in the weeds of trying to support either at this point. Just bare in mind that they will exist in the future. So lets remove it to keep this change clean.